### PR TITLE
Vdh agonizing flames

### PR DESCRIPTION
**Pull request by yajinni**
https://www.warcraftlogs.com/reports/JxyY7HCDcjqMA9tf/#fight=1&source=15

image

### CODE REVIEW

File: src/Parser/DemonHunter/Vengeance/Modules/Talents/AgonizingFlames.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        icon={<SpellIcon id={SPELLS.AGONIZING_FLAMES_TALENT.id} />}
+        value={`${this.owner.formatItemDamageDone(this.damage)}`}
+        label="Agonizing Flames"
+        tooltip={`This shows the extra dps that the talent provides.<br/>
+                  <b>Total extra damage:</b> ${formatNumber(this.damage)}`}
+      />
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(7);
+}
```
Comment (Pewtro) [inline src/Parser/DemonHunter/Vengeance/Modules/Talents/AgonizingFlames.js]: statisticOrder is deprecated - see here: https://github.com/WoWAnalyzer/WoWAnalyzer/commit/ece1bbeca0d3721ede078d256a30576faacb803d

File: src/Parser/DemonHunter/Vengeance/Modules/Talents/AgonizingFlames.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.AGONIZING_FLAMES_TALENT.id} />}
+        value={`${this.owner.formatItemDamageDone(this.damage)}`}
+        label="Agonizing Flames"
+        tooltip={`This shows the extra dps that the talent provides.<br/>
+                  <b>Total extra damage:</b> ${formatNumber(this.damage)}`}
+      />
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(7);
```
Comment (Gebuz) [inline src/Parser/DemonHunter/Vengeance/Modules/Talents/AgonizingFlames.js]: please use the new position prop in statisticBox instead:
https://github.com/WoWAnalyzer/WoWAnalyzer/blob/bf8e4dbf76569fc8d7c7c932bf5677889230489b/src/Parser/Paladin/Holy/Modules/Talents/CrusadersMight.js#L55

File: src/Parser/DemonHunter/Vengeance/Modules/Talents/AgonizingFlames.js (hunk 1 above)
Comment (Gebuz) [reply src/Parser/DemonHunter/Vengeance/Modules/Talents/AgonizingFlames.js]: oh man I'm too slow D:

**Comment by yajinni** [comment]
Made requested changes

File: src/Parser/DemonHunter/Vengeance/Modules/Talents/AgonizingFlames.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.AGONIZING_FLAMES_TALENT.id);
+  }
+
+  on_byPlayer_damage(event) {
+    const spellID = event.ability.guid;
+    if (spellID !== SPELLS.IMMOLATION_AURA_FIRST_STRIKE.id && spellID !== SPELLS.IMMOLATION_AURA_BUFF.id) {
+      return;
+    }
+        this.damage += calculateEffectiveDamage(event, PERCENT_BUFF);
```
Comment (MartijnHols) [inline src/Parser/DemonHunter/Vengeance/Modules/Talents/AgonizingFlames.js]: Please check your IDE settings for consistent indentation

**Pull request merged**